public/emoji/flags.css

```diff
@@ -0,0 +1,2069 @@
+/* glue: 0.13 hash: 69dcc6a805 */
+        .em-flags-1f6a9,
```

Comment:
^

Reply:
This is a generated file :slightly_smiling_face: